docs/guides/docs/intro-to-sway/rust-sdk.mdx

```diff
@@ -0,0 +1,323 @@
+---
+title: Rust Testing
+category: Intro to Sway
+parent:
+  label: Guides
+  link: /guides
+---
+
+## Testing the contract
+
+You can find the full contract code, along with example tests using the Rust SDK, in the provided repository.
+
+### Generating a Test Template in Rust
+
+To create your own test template using Rust, follow these steps with `cargo-generate`:
+
+1. Install `cargo-generate`:
+```bash
+cargo install cargo-generate
+```
+
+2. Generate the template:
+```bash
+cargo generate --init fuellabs/sway templates/sway-test-rs --name contract
+```
+
+### Running the Tests
```

Comment:
This section should be at the end of this page

Reply:
Good call